infrastructure/terraform/rafiki-test/cluster.tf

```diff
@@ -0,0 +1,12 @@
+
+module cluster {
+  source = "../modules/cluster"
```

Comment:
This references the cluster module defined in `infrastructure/terraform/modules/cluster/` and supplies the variables defined in `infrastructure/terraform/modules/cluster/variables.tf`.